### PR TITLE
feat(terra): add infra explanation page

### PR DESCRIPTION
**Pull request by madonuko**

### CODE REVIEW

File: pages/terra/infrastructure.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+name: Infrastructure (How does Terra work exactly?)
+description: The technical details behind the infrastructure and how different components work together.
+---
+
+# How Terra works
```
Comment (nothingneko) [inline pages/terra/infrastructure.md]: Title Case

"How Terra Works"

File: pages/terra/infrastructure.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+This document outlines the process of how a package ends up to the end users.
+
+## Terra installation
```
Comment (nothingneko) [inline pages/terra/infrastructure.md]: Title Case

"Terra Installation"

File: pages/terra/infrastructure.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+- API Interactive Docs: https://subatomic.fyralabs.com/docs/index.html
+
+### Tetsudou
+
+A metalink generator, so the clients (dnf/dnf5) knows what mirrors a repo has and which ones are preferred.
+
+- GitHub: https://github.com/terrapkg/tetsudou
+- URL: https://tetsudou.fyralabs.com/metalink?repo=terrarawhide
+
+## When CI builds a new package
```
Comment (nothingneko) [inline pages/terra/infrastructure.md]: Title Case

"When CI Builds a New Package"

File: pages/terra/infrastructure.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+1. The `autobuild.yml` workflow detects for changes in commits
+2. In the `manifest` stage, `anda ci` finds out what directories have been changed in the last commit
+3. A matrix of jobs will run for each architecture and each package
+4. GitHub Actions fetches the Terra builders
+5. It runs `anda build` with the Terra mock files
+6. It uploads built SRPMs and RPMs to Subatomic only if the build is successful and it's not in a pull request
+7. It notifies Madoguchi for a successful/failed build
+
+## When dnf/dnf5 needs a Terra package
```
Comment (nothingneko) [inline pages/terra/infrastructure.md]: Title Case

"When dnf/dnf5 Needs a Terra Package

File: pages/terra/index.mdx [hunk 5]
```diff
@@ -5,4 +5,10 @@ description: Welcome to the documentation for the Terra repository.
 
 # Welcome
 
-With Terra, you can install the latest packages knowing that quality and security are assured. Terra is also a great way to distribute your software on Fedora and its derivatives.
+Terra is a rolling repository RPM repository for Fedora Linux and its derivatives.
+With Terra, you can install the latest packages knowing that quality and security are assured.
+Terra is also a great way to distribute your software on Fedora and its derivatives.
+
+- Website: https://terra.fyralabs.com/
```
Comment (nothingneko) [inline pages/terra/index.mdx:12]: use md links instead?

File: pages/terra/infrastructure.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
+---
+name: Infrastructure (How does Terra work exactly?)
```
Comment (nothingneko) [inline pages/terra/infrastructure.md]: Title Case

"How Does Terra Work Exactly?"

File: pages/terra/infrastructure.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+## Terra installation
+
+There are two ways to install Terra:
+
+1. By installing the `terra-release` package (recommended). This makes sure your Terra installation
+   won't break even if we have any server migrations, infrastructure changes, etc.
+2. With `.repo` files. We provide `.repo` files at https://github.com/terrapkg/subatomic-repos.
+
+This document assumes Terra is installed via the 1st method. You should also look at the "Last
```
Comment (nothingneko) [inline pages/terra/infrastructure.md]: "first" instead of "1st"

**Review by nothingneko — CHANGES_REQUESTED** [review]
looks pretty good. please consider using markdown links instead of just raw links, it looks cleaner

File: pages/terra/index.mdx (hunk 5 above)
Comment (madonuko) [reply pages/terra/index.mdx:12]: The point is I want the links to be visible

Comment (nothingneko) [reply pages/terra/index.mdx:12]: alrightie

**Review by nothingneko — APPROVED** [review]

**Pull request merged**